docs/java/features/connectivity/destinations.md

```diff
@@ -18,6 +18,37 @@ The SAP Cloud SDK offers some basic functionality that helps with connecting to
 
 This concept is integrated with the [Destination Service](https://help.sap.com/viewer/cca91383641e40ffbe03bdc78f00f681/Cloud/en-US/7e306250e08340f89d6c103e28840f30.html) that is available on SAP Cloud Platform. If the application has a service binding to this service in place the SDK will provide access to these destinations.
 
+## Get a List of all Destination in Cloud Foundry  ##
+ 
+In order to fetch all destinations from the Destination Service you need to perform the following steps: 
+
+ 1) Invoke the `ScpCfDestinationLoader` with the `DestinationService` and `XsuaaService` instances.
+
+```java
+ final ScpCfDestinationLoader destinationLoader = new ScpCfDestinationLoader(destinationService, xsuaaService);
+```
+
+ 2) Build a `DestinationOptions` object based on the retrieval strategy for your particular use case. The Retieval strategy can be `ALWAYS_SUBSCRIBER`, `ALWAYS_PROVIDER` or `SUBSCRIBER_THEN_PROVIDER`. E.g. for `SUBSCRIBER_THEN_PROVIDER` use:
```

Comment:
```suggestion
 2) For the upcoming call build a `DestinationOptions` object. It contains the configuration on how the destination service is being queried. If you have a simple application without provider/subscriber setup, then your initial configuration may look like this:
    ```java
    // in case of Subscriber Tenant
    final DestinationOptions options = DestinationOptions.builder().build();
    ```
  If you have a provider/subscriber setup, a retrieval strategy must be chosen according to your particular use case. The retrieval strategy can be `ALWAYS_SUBSCRIBER`, `ALWAYS_PROVIDER` or `SUBSCRIBER_THEN_PROVIDER`. E.g. for `SUBSCRIBER_THEN_PROVIDER` use:
```